plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""The entropy calculation implementation."""
+
+from __future__ import division
+from __future__ import unicode_literals
+
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
+  """Calculates the byte entropy of input files."""
+
+  NAME = 'entropy'
+  DESCRIPTION = 'Calculates the entropy of input data.'
+
+  def __init__(self):
+    """Initializes the entropy hasher."""
+    super(EntropyHasher, self).__init__()
+    self._counter = collections.Counter()
+    self._length = 0
+
+  @classmethod
+  def GetAttributeName(cls):
+    """The attribute name for the result."""
+    return 'file_entropy'
```

Comment:
Make this a class constant?

Reply:
Leaving this as is, as this isn't reused anywhere, and would just add extra indirection.

Reply:
If we are discussing removing indirection. I opt:

1. move the attribute name into a class constant
2. remove GetAttributeName

Reply:
Done